regression/tests/studio/test_studio_textbooks.py

```diff
@@ -36,40 +36,34 @@ def setUp(self):
             self.course_info['run'])
 
         self.textbook_page.visit()
-        self.textbook_name = 'book_{}'.format(uuid4().hex)
-        self.chapter_name = 'chap_1'
-        # Add textbook
-        self.add_textbook()
 
-    def add_textbook(self):
+    def test_textbook_crud(self):
         """
-        Add textbook
+        Verifies that user can add, verify, edit and delete textbook
         """
+        # Add textbook
+        textbook_name = 'book_{}'.format(uuid4().hex)
+        chapter_name = 'chap_1'
         self.textbook_page.open_add_textbook_form()
         self.textbook_page.set_input_field_value(
-            '.edit-textbook #textbook-name-input', self.textbook_name
+            '.edit-textbook #textbook-name-input', textbook_name
         )
         self.textbook_page.set_input_field_value(
-            '.edit-textbook #chapter1-name', self.chapter_name
+            '.edit-textbook #chapter1-name', chapter_name
         )
         self.textbook_page.upload_textbook('test_pdf.pdf')
-        self.textbook_page.click_textbook_submit_button()
 
-    def test_textbook_verify_on_lms(self):
-        """
-        Verifies that user can add and visit textbook on LMS
-        """
-        self.add_textbook()
+        # Verify the added textbook on LMS
```

Comment:
nit: this comment should be 1 line down, after the "click_textbook_submit_button()" line.